### ISSUE TITLE
Postponing tasks doesn't actually postpone the task

### ISSUE THREAD

**Issue by SuperJappie08** [VladimirMarkelov/ttdl#62]
Hi,

I noticed when I tried to postpone a task, that the due date doesn't change in the file.
It also only updates the `due` tag when passing `--dry-run`.

Example outputs below:
```
~ > ttdl postpone 34 1d --dry-run
Todos to be postponed:
 # D P Due        Subject
--------------------------
34     2022-10-10 Make Issue about postpone on git due:2022-10-10

New todos:
34     2022-10-11 Make Issue about postpone on git due:2022-10-11
--------------------------
1 todos (of 39 total)


~ > ttdl postpone 34 1d
Changed todos:
 # D P Due        Subject
--------------------------
34     2022-10-11 Make Issue about postpone on git due:2022-10-10
--------------------------
1 todos (of 39 total)
```

I think the logic for this functionality is there in `--dry-run` mode, since the `due:` tag actually gets updated.

EDIT:
Platform: Windows 10
Installed via: cargo
Version of ttdl: 3.4.4

**Comment by VladimirMarkelov** [comment]
Fixed in `3.4.5`: https://github.com/VladimirMarkelov/ttdl/commit/16b8670d357a89909a147a29cd603fda2928157a#diff-42cb6807ad74b3e201c5a7ca98b911c5fa08380e942be6e4ac5807f8377f87fc

**Comment by VladimirMarkelov** [comment]
3.4.5 published on crates.io
